sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateKeyUsage.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Security.KeyVault.Certificates
+{
+    /// <summary>
+    /// Supported usages of a certificate key
+    /// </summary>
+    public readonly struct CertificateKeyUsage : IEquatable<CertificateKeyUsage>
```

Comment:
>CertificateKeyUsage [](start = 27, length = 19)

I don't know that we need to change the name of this.  I don't believe it collided with any types from the Keys namespace. In the Keys namespace there is a somewhat similar type but it's named KeyOperation

Reply:
I thought it would make it more consistent within the namespace, but happen to change back. It just seemed odd that every other type was prefaced with "Certificate" except this one. Might be less approachable if devs were more used to typing "Certificate" before commonly used types with `CertificateClient`. Thoughts?